src/kdc/kdc_util.c

```diff
@@ -1073,20 +1073,22 @@ void limit_string(char *name)
 static krb5_error_code
 enctype_name(krb5_enctype ktype, char *buf, size_t buflen)
 {
-    char *name;
+    const char *name, *deprecated = "";
```

Comment:
nit: since this can take values other than "DEPRECATED:", I'd prefer a different name - perhaps `status` or `support` or somesuch?

Reply:
Sure, I've changed it to `prefix`.